src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  justify-content: space-between ;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    display: flex;
+    text-decoration: none;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+
+    @extend %larg-text;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 60px;
+    text-transform: uppercase;
+    color: #060b35;
+    text-decoration: none;
+
+    &:hover {
+      color: #00acdc;
+      line-height: 60px;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+  display: block;
+  color: #00acdc;
+  line-height: 60px;
+}
+
+.is-active::before {
+  content: "";
+  position: absolute;
+  display: block;
+  width: 100%;
+  bottom: 0;
+  border-bottom: 4px solid #00acdc;
+  border-radius: 8px;
+}
```

Comment:
Move this part to `blocks/nav.scss` - it's not a part of `header`-block.
Don't forget to connect style to `main`.